File: src/#1029/solution.go
package solution

import "sort"

func twoCitySchedCost(costs [][]int) int {
	total := 0
	dArr := []int{}
	for _, c := range costs {
		total += c[0]
		dArr = append(dArr, c[1]-c[0])
	}

	sort.Ints(dArr)

	for i := 0; i < len(dArr)/2; i++ {
		total += dArr[i]
	}
	return total
}

File: src/#1029/solution.js
var twoCitySchedCost = function(costs) {
    let total=0
    let dArr=[]

    costs.forEach(c => {
        total+=c[0]
        dArr.push(c[1]-c[0])
    });
    
    dArr.sort((a,b)=>a-b)
    for(let i=0;i<dArr.length/2;i++){
        total+=dArr
    }
    return total
};